tools/mob_suite/mob_recon.xml

```diff
@@ -17,7 +20,17 @@
     #if str($adv_param.min_length_condition.min_length_param) == "True":
       --min_length ${adv_param.min_length_condition.min_length_value}
     #end if 
-    --run_typer --outdir '.';
+    --run_typer --min_rep_evalue '${adv_param.min_rep_evalue}'
+    --min_rep_evalue '${adv_param.min_rep_evalue}'
+    --min_mob_evalue '${adv_param.min_mob_evalue}'
+    --min_con_evalue '${adv_param.min_con_evalue}'
+    --min_rep_ident '${adv_param.min_rep_ident}'
+    --min_mob_ident '${adv_param.min_mob_ident}'
+    --min_con_ident  '${adv_param.min_con_ident}'
+    --min_rpp_ident   '${adv_param.min_rpp_ident}'
+    --outdir '.' && 
+    mkdir ./sequences && cp plasmid*.fasta chromosome.fasta  ./sequences 2> /dev/null | true 
```

Comment:
Yes, this is intended behaviour. Sometimes `chromosome.fasta` file is not produced by the mob-recon (i.e. the input sequences are all plasmids). In that case the  copy command will fail. Probably I could of written some python code to check for the file existence, but I feel this simple solution will work for now. If the file does not exist it will create an empty `chrosmosme.fasta` file in `sequences` folder. Any suggestions to do it in more nicer way using the Cheetah code?

Reply:
Okay, this makes sense.  I think it would be a bit easier to understand if you changed to `(cp plasmid*.fasta chromosome.fasta  ./sequences 2> /dev/null || true)`.  That is `||` is for a logical OR, while `|` is for redirecting stdout (and `()` wraps everything in a subshell so that the exit code will only default to `true` if the copy command fails).

I'm not sure if there's a way to do this in Cheetah code since I'm assuming the files are only generated after `mob_recon` is run (and so after the Cheetah code is run to produce the commands).

Reply:
I've tried to look at the solution and the only one is to use a python script to do a post-run manipulations including copying of files. Cheetah code forms a command to run in bash at the execution time and no further logic is possible. I've made a change in the `cp` directive. Thank you!

Reply:
Thanks for making that change.  It all looks good.